nixos/platform/default.nix

```
@@ -87,6 +87,19 @@ in {
       description = "Where to find the ENC services json file.";
     };
 
+    flyingcircus.hostRgwAddress = mkOption {
+      default = null;
+      type = with types; nullOr str;
+      description = ''
+        IP address for the radosgw object storage proxy running on the
+        virtualization host on port 7480. It allows VMs to access the
+        S3(-compatible) storage via the fast storage network.
+
+        This is the same as the `rgw.local` entry in `/etc/hosts`. Set if the
```

foo

---

```suggestion
        This is the same as the `rgw.local` entry in `/etc/hosts`. Value is set if the
```

Clarify that *Set* describes the state of the option value, not the verb/ action of the possibility that you can/ should set it for virtual machines.

If we decide to mark it as read-only, possibly even extend to *Value is set by the platform if…*